### PR TITLE
Move properties out of custom.properties file into application.proper…

### PR DESCRIPTION
**Pull request by FeimeiChen**
# Ticket Link

[Ticket 1487](https://uhawaii.atlassian.net/browse/GROUPINGS-1487)

# List of squashed commits

- Commit 1: Move properties out of custom.properties file into application.properties file
- Commit 2: remove the line

# Test Checklist

- [x] Unit Tests Passed:
- [x] Jasmine Tests Passed:
- [x] General Visual Inspection:

### CODE REVIEW

**Review by mhodgesatuh — APPROVED** [review]

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -24,6 +25,20 @@ cas.saml.tolerance=180000
 cas.send.renew=false
 
 # =========================================================================
+app.iam.request.form=https://www.hawaii.edu/bwiki/display/UHIAM/UH+Groupings+Request+Form
```
Comment (JorWo) [inline src/main/resources/application.properties:28]: As stated in the API PR, you can remove this line.

Comment (FeimeiChen) [reply src/main/resources/application.properties]: done

**Review by JorWo — APPROVED** [review]
Looks good!

**Comment by fduckart** [comment]
I thought we had already removed that AppConfig class, but maybe I imagined that. We shouldn't need that class now, if the overrides via the main properties files is working correctly. Is there some edge case that requires it or something?

**Comment by mhodgesatuh** [comment]
# Is there some edge case that requires it or something?

not to my knowledge

**Comment by JorWo** [comment]
I will create a ticket to remove the AppConfig class.

**Pull request merged**